core/testsuite/src/test/java/com/blazebit/persistence/testsuite/SizeTransformationTest.java

```diff
@@ -356,4 +359,40 @@ public void testSizeTransformationMultiLevelCorrelatedFromClause() {
             cb.getResultList();
     }
 
+    // from issue #513
+    @Test
```

Comment:
Some DBs seem to have a problem with that expression:

* DB2 - https://travis-ci.org/Blazebit/blaze-persistence/jobs/374092236#L1515
* SQL Server - https://travis-ci.org/Blazebit/blaze-persistence/jobs/374092224#L1537

We need to investigate if there is a way to make it work, otherwise we will ignore the test for the problematic DBMS